src/WebJobs.Script/Description/FunctionLogger.cs

```diff
@@ -67,25 +67,35 @@ public TraceWriter CreateUserTraceWriter(TraceWriter traceWriter)
         }
 
         // Helper to emit a standard log message for function started.
-        public void LogFunctionStart(string invocationId)
+        public void LogFunctionStart(string functionName, string functionInvocationId)
         {
-            string startMessage = $"Function started (Id={invocationId})";
-            TraceWriter.Info(startMessage);
+            string startMessage = $"Function started (Id={functionInvocationId} Name={functionName})";
```

Comment:
If you rebase, you'll see that I'm now already adding function name to the log statements ([this PR](https://github.com/Azure/azure-functions-host/pull/2454)). Adding this one here would be unecessary and too much I think.